all_test.go

```diff
@@ -17,6 +18,10 @@ func Test_Must(t *testing.T) {
 	})).ToBe("01:37:58\n\n")
 }
 
+func removeWhitespace(s string) string {
+	return strings.TrimSpace(strings.Replace(s, " ", "", -1))
```

Comment:
In my experiments I found that tesseract can be quite unpredictable about spaces (i.e. how many there are in a gap between two characters) - between versions at least.

My thinking was that for these tests what really matters is that we only get characters from the whitelist back; how many spaces tesseract decides exist between two characters shouldn't really be a concern of gosseract.